### ISSUE TITLE
Passing dockerfile path from Git as parameter

### ISSUE THREAD

**Issue by sharadpattanshetti** [hadolint/hadolint#447]
Wanted to know if it is possible to pass the dockerfile path from git repo as parameter to hadolint?

**Comment by zemanlx** [comment]
I do not think so but you can pipe it 

```bash
wget -O- https://my.git/Dockerfile | hadolint -
```

**Comment by sharadpattanshetti** [comment]
Thanks @zemanlx . I tried executing below command 
docker run --rm -i hadolint/hadolint  wget -O- https://my.git/Dockerfile | hadolint

but it doesnt work.

**Comment by zemanlx** [comment]
`wget` needs to be installed in your environment (which it usually is). It downloads your `Dockerfile` and passes it to `hadolint` (in your case running in docker).

Here is an example of running it for `redis` image

```bash
wget -qO- https://raw.githubusercontent.com/dockerfile/redis/master/Dockerfile |  docker run --rm -i hadolint/hadolint hadolint - 
```

you should see the following output

```bash
/dev/stdin:8 DL3006 Always tag the version of an image explicitly
/dev/stdin:11 SC2035 Use ./*glob* or -- *glob* so names with dashes won't become options.
/dev/stdin:11 DL3003 Use WORKDIR to switch to a directory
```

**Comment by sharadpattanshetti** [comment]
Thanks @zemanlx , I tried it on Windows PowerShell, it gives below error
Invoke-WebRequest : A parameter cannot be found that matches parameter name 'qO-'.
At line:1 char:6

If i remove the -qO- option then it does output as below for my docker file: 
/dev/stdin:1:1 unexpected carriage return expecting '#', ADD,

**Comment by zemanlx** [comment]
@sharadpattanshetti sorry to hear that it is not working in Windows. I am afraid I do not have any Windows to help you. My advice above should work for Linux/MacOS devices.

Maybe there will be somebody else who can respond to this issue.
